### ISSUE TITLE
Ability to use filtered WFS in webgeom POST

### ISSUE THREAD

**Issue by jread-usgs** [USGS-R/geoknife#287]
see comments on #277 

From Blodg:
Something to note though, you don't actually have to do the gmlid lookup if you have implemented filtering like this. The GDP can take the filtered WFS request rather than a list of gmlids. You do have to include `<wfs:PropertyName>the_geom</wfs:PropertyName>` for geoserver-shapefile WFS and possibly other names for the geometry column for other WFS implementations, but you could skip the gmlid lookup part.

**Comment by jread-usgs** [comment]
I thought we did this for v1.3.5 ...?

**Comment by dblodgett-usgs** [comment]
Yeah. This looks to be done here. https://github.com/USGS-R/geoknife/blob/ee363abdf0a884d33287430c2460e70b1697677e/R/values-webgeom.R#L67

Looking at one of the vignette webgeom examples, I see this is what posts to geoserver:
```
<wfs:GetFeature xmlns:ogc="http://www.opengis.net/ogc" xmlns:wfs="http://www.opengis.net/wfs" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:gml="http://www.opengis.net/gml" xmlns:ows="http://www.opengis.net/ows/1.1" service="WFS" version="1.1.0" xsi:schemaLocation="http://www.opengis.net/wfs http://schemas.opengis.net/wps/1.0.0/wpsExecute_request.xsd">
  <wfs:Query typeName="sample:simplified_huc8">
    <ogc:PropertyName>HUC_8</ogc:PropertyName>
    <ogc:Filter>
      <ogc:Or>
        <ogc:PropertyIsEqualTo matchCase="true">
          <ogc:PropertyName>HUC_8</ogc:PropertyName>
          <ogc:Literal>09020306</ogc:Literal>
        </ogc:PropertyIsEqualTo>
        <ogc:PropertyIsEqualTo matchCase="true">
          <ogc:PropertyName>HUC_8</ogc:PropertyName>
          <ogc:Literal>14060009</ogc:Literal>
        </ogc:PropertyIsEqualTo>
      </ogc:Or>
    </ogc:Filter>
  </wfs:Query>
</wfs:GetFeature> 
```

That same `ogc:Filter` can be passed into the wfsquery without specifying gmlids. So maybe this is half done? It (the same filter used in webGeom) just needs to be implemented here: https://github.com/USGS-R/geoknife/blob/master/R/processInputsToXML.R#L137 ... I think. @jiwalker-usgs and @jread-usgs is this making sense?

**Comment by dblodgett-usgs** [comment]
I've got this implemented. The only thing that has changed is that this test fails: https://github.com/USGS-R/geoknife/blob/master/tests/testthat/test-geoknife_setters.R#L61 because we aren't hitting the service and checking to see if the values are valid. Seems to work pretty good, but we'll want to play with it some more.

**Comment by dblodgett-usgs** [comment]
fixed in #364
